src/main/java/org/jenkinsci/test/acceptance/po/WizardCustomizeJenkins.java

```diff
@@ -94,8 +94,26 @@ public void startInstall() {
         clickButton("Install");
     }
 
-    public void deselectAll() {
+    public void selectAll() {
+        clickLink("All");
+        elasticSleep(200);
+    }
+
+    public void selectNone() {
         clickLink("None");
         elasticSleep(200);
     }
+
+    public void selectSuggested() {
+        clickLink("Suggested");
+        elasticSleep(200);
+    }
```

Comment:
wait for the number of selected plugins to change?  (this assumes that a test would not click on this twice, but well that would be silly right!)?